### PR TITLE
SuntContent Bid Adapter : Initial Release

### PR DESCRIPTION
**Pull request by sag-henmus**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter 
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- contact email of the adapter’s maintainer: [tech@seeding-alliance.de](mailto:tech@seeding-alliance.de)
- test parameters for validating bids:
```
{
  bidder: 'suntContent',
  params: {
    url:"https://www.pubdemo.suntcontent.com/",
    adUnitId: "ba4g"
  }
}
```

### CODE REVIEW

**Comment by sag-henmus** [comment]
Hi everyone, don't want to be pushy, but this PR is now nearly two months old. Any updates?

**Comment by lksharma** [comment]
Hi @sag-henmus, Thanks for reaching out and apologies for the delay. I'll prioritize this PR today and let you know if any updates are needed.

File: modules/suntContentBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+'use strict';
+
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER, NATIVE} from '../src/mediaTypes.js';
+import {_map, deepSetValue, isArray, isEmpty, replaceAuctionPrice} from '../src/utils.js';
+import {config} from '../src/config.js';
+import {convertOrtbRequestToProprietaryNative} from '../src/native.js';
+
+const GVL_ID = 1097;
+const BIDDER_CODE = 'suntContent';
```
Comment (lksharma) [inline modules/suntContentBidAdapter.js:11]: bidder code should be lower case alphanumeric

File: modules/suntContentBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+    return bidRequests
+      .map((bidRequest, id) => {
+        const bidResponse = bidResponses[id];
+
+        const type = bidRequest.nativeParams ? NATIVE : BANNER;
+
+        if (bidResponse) {
+          const bidObject = {
+            requestId: bidRequest.bidId, // TODO get this value from response?
```
Comment (lksharma) [inline modules/suntContentBidAdapter.js:163]: Is this something you still intend to update?

**Comment by lksharma** [comment]
@sag-henmus, I wasn't able to see the accompanying docs PR. Could you please additionally create a pull request for the documentation at: https://github.com/prebid/prebid.github.io

**Comment by sag-henmus** [comment]
@lksharma thanks for the review. A docs PR was already created with the accompanying server adapter, see https://github.com/prebid/prebid.github.io/pull/4076. The biddercode suntContent was used there as well, hence the same one here. Concerning the TODO, for us this is fine ATM.

**Comment by lksharma** [comment]
@sag-henmus, in line with the requirements for PBJS adapters, the biddercode must be lowercase. Given that this is the initial release, would you be able adjust the biddercode to adhere to these standards? Additional details can be found [here](https://docs.prebid.org/dev-docs/bidder-adaptor.html#:~:text=The%20bidder%20code%20must%20be%20lowercase%20alphanumeric.%20The%20only%20special%20character%20allowed%20is%20underscore.). This also would require you to update your documentation to reference the correct biddercode. To make sure you changes are up to date can you also rebase your changes from latest master.

**Comment by sag-henmus** [comment]
@lksharma given that the prebid server adapter unfortunately is already in active use and in camelCase, I hope you can see why I'd prefer not to change it to lowercase.

**Comment by lksharma** [comment]
@patmmccann/@ChrisHuie, could you share your thoughts on this? Right now, the bidderCode `suntContent` doesn't quite align with the requirements for allowed bidder codes in adapters.

**Comment by lksharma** [comment]
@sag-henmus Any updates on the suggested changes?

**Comment by sag-henmus** [comment]
@lksharma Sorry for the delay, with the release of prebid server 2.0 we will change the adapter to lowercase.

**Comment by patmmccann** [comment]
closing as stale

**Pull request closed without merging**